rplugin/python3/deoplete/sources/beancount.py

```diff
@@ -35,18 +39,29 @@ def get_complete_position(self, context):
         return m.start() if m else -1
 
     def gather_candidates(self, context):
+        attrs = self.attributes
         if re.match(r'^\d{4}[/-]\d\d[/-]\d\d \w*$', context['input']):
             return [{'word': x, 'kind': 'directive'} for x in DIRECTIVES]
+        # line that starts with whitespace (-> accounts)
+        if re.match(r'^(\s)+\w+$', context['input']):
```

Comment:
I specifically set smartcase and ignorecase in my vimrc so that lowercase matches work so I really want to keep that functionality.